src/test/java/TestMapCreator.java

```diff
@@ -0,0 +1,88 @@
+import collections.MapCreator;
+import org.junit.jupiter.api.AfterEach;
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.Test;
+
+import java.io.ByteArrayOutputStream;
+import java.io.PrintStream;
+import java.util.Arrays;
+import java.util.List;
+import java.util.TreeMap;
+
+import static org.junit.jupiter.api.Assertions.assertEquals;
+
+public class TestMapCreator {
+
+    private final PrintStream standardOut = System.out;
+    private final ByteArrayOutputStream outputStreamCaptor = new ByteArrayOutputStream();
+
+    List<String> myTestStrings = Arrays.asList("foo", "bar", "delta");
+    List<Integer> myTestNumbers = Arrays.asList(1, 2, 3, 4);
+    TreeMap<String, Integer> testMapKV_1 = new TreeMap<String, Integer>() {{
```

Comment:
Please use interfaces (like you already did for Set and List) for Maps as well throughout tests

Reply:
done